packages/edit-site/src/components/block-editor/site-editor-canvas.js

```diff
@@ -68,7 +75,10 @@ export default function SiteEditorCanvas() {
 				) : (
 					<BlockTools
 						className={ classnames( 'edit-site-visual-editor', {
-							'is-focus-mode': isFocusMode || !! editorCanvasView,
+							'is-focus-mode':
+								isFocusMode ||
+								!! editorCanvasView ||
+								isEditingTemplate,
```

Comment:
What's the reason for the change here and why it's limited only to the site editor?